### PR TITLE
Log queries that time out (Fixes #8656)

### LINKED ISSUE #8656: Log timeout queries
### Feature Request

Similar to #6429, will be useful to log which queries have crossed the ``query-timeout`` parameter and killed.

### PR DESCRIPTION
**Pull request by drewblas**
Closes #8656 

###### Required for all non-trivial PRs
- [x] Rebased/mergable
- [ ] Tests pass
- [x] CHANGELOG.md updated
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)
- [x] Config changes: update sample config (`etc/config.sample.toml`), server `NewDemoConfig` method, and `Diagnostics` methods reporting config settings, if necessary
- [ ] [InfluxData Documentation](https://github.com/influxdata/docs.influxdata.com): issue filed or pull request submitted \<link to issue or pull request\>

### CODE REVIEW

**Comment by drewblas** [comment]
The tests pass for me locally, but appveyor seems to crash before even starting them and I don't know why

File: coordinator/config.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ type Config struct {
 	WriteTimeout         toml.Duration `toml:"write-timeout"`
 	MaxConcurrentQueries int           `toml:"max-concurrent-queries"`
 	QueryTimeout         toml.Duration `toml:"query-timeout"`
+	LogQueryTimeouts		 bool					 `toml:"log-query-timeouts"`
```
Comment (e-dard) [inline coordinator/config.go]: Your code has not been ran through `gofmt`. We can't accept it until it passes `gofmt` as well as `go vet`. Our CI will ensure that `go vet` passes.

File: query/task_manager.go [hunk 2]
```diff
@@ -266,6 +270,10 @@ func (t *TaskManager) waitForQuery(qid uint64, interrupt <-chan struct{}, closin
 
 		t.queryError(qid, err)
 	case <-timerCh:
+		if(t.LogQueryTimeouts){
```
Comment (e-dard) [inline query/task_manager.go]: `gofmt` again. Go does not use parentheses for single expression `if` statements.

**Comment by drewblas** [comment]
@e-dard Sounds good.  I've removed the config as requested and there are no formatting errors now.

**Comment by MickaelBergem** [comment]
@drewblas there seems to be a conflict on the CHANGELOG.md file. Maybe triggering a new build will magically solve the AppVeyor issue?

**Comment by e-dard** [comment]
@drewblas if you wanted to revisit this we could probably merge it for 1.8.

**Comment by azhurbilo** [comment]
@e-dard what status of this critical for real production usage feature?
do you plan implement timeout logging in 2.x release or you plan release 1.8 ?

**Pull request closed without merging**